kernelci/kernelci-backend
# Add LAVA callback parsing tests

This commit covers basic use cases for LAVA callback parsing.
Test cases covered:
- login test case document should be added if "login-action or
"auto-login-action" is present in lava test suite
- Boot log lines should never appear in the test case document
- login test case status should be set to "FAIL" when there is an
unhandled exception during boot even if login-action/auto-login-action
status is "PASS"
- Checks if log lines are properly extracted and attached for a specific test case
